spec/controllers/api/v3/blood_pressures_controller_spec.rb

```diff
@@ -1,36 +1,33 @@
 require "rails_helper"
 
 RSpec.describe Api::V3::BloodPressuresController, type: :controller do
+  before { Flipper.enable(:regions_prep) }
```

Comment:
For this sort of wide-reaching feature flag, we will want to be testing **both** code paths for awhile to ensure we are able to turn the flag off for an easy revert after shipping.  It wouldn't surprise me to see the feature flagged code live on into January given holidays and also the fact that mobile rollout and resync takes time.

Something like this will work for the entire controller spec:

```ruby
[:enable, :disable].each do |flag_toggle|
  RSpec.describe "Api::V3::BloodPressuresController with regions_prep #{flag_toggle}", type: :controller do
    before { Flipper.public_send(flag_toggle, :regions_prep) }
...
```

Reply:
Thanks @rsanheim! These specs are a bit tricky since they only partially depends on the flag. We've been thinking about the `regions_prep` flag as something that is extremely short-term. In fact, we're looking to raise a PR to remove it this week itself. The `region_level_sync` flag is what will stick around long-term (across months), this `regions_prep` was a super transitionary flag, the quicker we get rid of the easier it is for everyone.

This is largely why we've avoided massive structural spec changes just to support this flag (more or less everywhere), the mental overhead and the overall cost just doesn't justify the upkeep.

However, using your above snippet (and without too much headache) we managed to get this working with a slight modification to avoid running the region specs when the `regions_prep` isn't enabled. It's covered [here](https://github.com/simpledotorg/simple-server/pull/1333/commits/f343427699cf80a265bfc15ae0902876e78cd5d1?diff=unified&w=1). However, now our specs take 10m to run on CI :)

Let us know if that's cool.